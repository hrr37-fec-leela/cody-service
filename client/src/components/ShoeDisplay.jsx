import React from 'react';
import styles from './Styles.module.css';

const ShoeDisplay = (props) => {
  return (
    <div className={styles.shoeDisplayMain}>
      <img
        className={styles.mainShoeImage}
        src={props.mainDisplay}
        alt="CurrySix-main"
      />
      {/* <video
        className="main-shoe-image"
        src="https://s3-us-west-2.amazonaws.com/hrr37-fec-underarmour-images/CurrySix-405/CurrySix360-405.mp4"
        controls loop>
      </video> */}
    </div>
  );
};

export default ShoeDisplay;