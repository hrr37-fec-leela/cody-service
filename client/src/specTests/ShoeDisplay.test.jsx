import React from 'react';
import ShoeDisplay from '../components/ShoeDisplay.jsx';
import { shallow, mount, render } from 'enzyme';

describe('<ShoeDisplay />', () => {

  it('no unintentional changes to ShoeDisplay component', () => {
    const ShoeDisplayComponent = shallow(<ShoeDisplay />);
    expect(ShoeDisplayComponent).toMatchSnapshot();
  });

});