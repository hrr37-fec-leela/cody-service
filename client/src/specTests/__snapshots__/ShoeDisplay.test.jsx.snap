// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShoeDisplay /> no unintentional changes to ShoeDisplay component 1`] = `ShallowWrapper {}`;
